docs/usage/ippool-affinity-node.md

```diff
@@ -1,96 +1,63 @@
 # SpiderIPPool node affinity
 
-*Spiderpool supports multiple ways to select ippool. Pod will select a specific ippool to allocate IP according to the corresponding rules that with different priorities. Meanwhile, ippool can use selector to filter its user.*
+*Spiderpool supports configuring the affinity between IP pools and Nodes, which enables only Pods running on specific Nodes to use the IP pools that be affinity to these Nodes. Node affinity should be regarded as a **filtering mechanism** rather than a [pool selection rule](TODO).*
 
 ## Setup Spiderpool
 
 If you have not set up Spiderpool yet, follow the guide [Quick Installation](https://github.com/spidernet-io/spiderpool/blob/main/docs/usage/install.md) for instructions on how to install and simply configure Spiderpool.
 
 ## Get started
 
-After understanding what selectors' "filtering mechanism" is, let's take a look at how selectors work with ippool selection rules. There are two ippools:
+Since my cluster has only two Nodes (1 master and 1 worker), I have to remove the relevant taints on master Node through `kubectl taint`, so that ordinary Pods can also be scheduled to it. If your cluster has two or more worker Nodes, please ignore the step above.
```

Comment:
```suggestion
Since the cluster in this example has only two nodes (1 master and 1 worker), it is required to remove the relevant taints on the master node through `kubectl taint`, so that ordinary pods can also be scheduled to it. If your cluster has two or more worker nodes, please ignore the step above.
```
Normally, we need not to capitalize Pod and Node unless it's special in a code snippet.

Reply:
其实一般 K8s 的资源在文档中都应该使用 Kind 名称（参见 K8s 官方文档，均不会全小写资源）。上文中的 "Node" 替换为 "node" 其实还说的过去，其含义就由 "资源 Node" 变成了一般语言中的 "节点"，而将 "Pods" 改掉其实就不太好了。

Reply:
嗯嗯，如果是引用代码中的内容，可以加个 `Pods`